### PR TITLE
Form view

### LINKED ISSUE #20: implement view for submission page
Implement a submission form containing: 
- [x] Founder name 
- [x] Startup Name (should be unique)
- [x] Phone number
- [x] E-mail
- [x] Field (drop down list)
- [x] Startup stage (drop down list)
- [x] photo (e.g startup logo)

### PR DESCRIPTION
**Pull request by Karyum**
close #20

### CODE REVIEW

**Comment by Suuuuha** [comment]
you reckon we should keep the classes names??

**Comment by Karyum** [comment]
any reason why we should remove them ??? @Suuuuha

**Comment by Suuuuha** [comment]
because we're gonna use them for tachyons only (i think)

**Comment by Karyum** [comment]
@Suuuuha  changes were made

File: src/views/startup_submission.hbs [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<div class="">
+  <h1>Submit your own startup!</h1>
+</div>
+
+<div class="">
+  <form class="" action="/postData" method="post">
```
Comment (esraajb) [inline src/views/startup_submission.hbs]: maybe instead of postData in the action, use 'registerStartup'

File: src/views/startup_submission.hbs [hunk 2]
```diff
@@ -0,0 +1,76 @@
+<div class="">
+  <h1>Submit your own startup!</h1>
+</div>
+
+<div class="">
```
Comment (esraajb) [inline src/views/startup_submission.hbs]: probably wont need this div

**Comment by Karyum** [comment]
@esraajb that because the label would be bound to the input so the 'for' attribute on the label is the same as the id of the input
https://stackoverflow.com/questions/18432376/what-does-for-attribute-do-in-html-label-tag

**Pull request merged**